tests/test_datetime.py

```diff
@@ -54,6 +54,24 @@ def test_nested_context_manager(datetime_function):
     assert datetime_function() != dt2
 
 
+@pytest.mark.parametrize('datetime_function', [datetime.utcnow, datetime.now])
+def test_start_stop(datetime_function):
+
+    dt = datetime(2016, 1, 1, 13, 54)
+    assert datetime_function() != dt
+
+    spell = immobilus('2016-01-01 13:54')
+    assert datetime_function() != dt
+
+    try:
```

Comment:
Why do you use here try-finally?

Reply:
If the assertion fails, then an `AssertionError` is raised and `spell.stop` would never be called without the `finally` clause. That would leave immobilus active when the next test starts, when it should be inactive.

Because the test is passing, we don't strictly need this, but if it did fail one day it might cause furthre failures in subsequent tests that would be confusing. Best to isolate the test.

Reply:
I realised. Thanks for explanation and for PR.

Reply:
No problem. Thank you for this project and being so responsive to these PRs. I am almost done with swapping out freezegun for this, I now just have one problem with the `fake_mktime` because `time.tzset` which it uses is not available on windows. I'm not quite sure what the proper fix would look like yet, but I know that the `test_mktime.py` tests both pass fine if we simply avoid faking `mktime` on windows and just `return original_mktime(timetuple)`. Maybe we don't have to  fake this one at all? I've read through #9 but I'm still a little unsure of expected behaviour here. I will need to get setup on a Linux machine too and play with this a little to understand I think.

Thanks again.

Reply:
I'm going to tune travis to run tests on several operating systems.

Reply:
I've fixed mktime on windows and added test plans for Windows. Please, try out version 0.2.15 from pypi.

Reply:
Thanks @pokidovea! Your fixes and quick releases are very much appreciated 👍 

Unfortunately, when I use 0.2.15 for the unit tests of my project, I now have some other problems. Please see #19